### PR TITLE
DOMPDF related security bugs, language control system and session handling

### PR DESCRIPTION
**Pull request by filippolauria**
Hello,
after having chmodded all files to 644 and directories to 755, I replaced DOMPDF library 0.5.1 with 0.6.2 both for the operator and for users.
I also have created a cleaner way to handle language files, simplified session handling, and login/logout functionalities.

### CODE REVIEW

File: dologin.php [hunk 1]
```diff
@@ -13,73 +13,78 @@
...
+$operator_id = $row['id'];
+
+if (array_key_exists('operator_login_error', $_SESSION)) {
+    unset($_SESSION['operator_login_error']);
+}
+$_SESSION['daloradius_logged_in'] = true;
+$_SESSION['operator_user'] = $operator_user;
+$_SESSION['operator_id'] = $operator_id;
+
+// lets update the lastlogin time for this operator
```
Comment (filippolauria) [inline dologin.php]: include('library/opendb.php'); should be added here

File: dologin.php [hunk 2]
```diff
@@ -13,73 +13,78 @@
...
+$now = date("Y-m-d H:i:s");
+$sqlFormat = "update %s set lastlogin='%s' where username='%s'";
+$sql = sprintf($sqlFormat,
+    $configValues['CONFIG_DB_TBL_DALOOPERATORS'], $now, $operator_user);
+$res = $dbSocket->query($sql);
+header('Location: index.php');
```
Comment (filippolauria) [inline dologin.php]: include('library/closedb.php'); should be included here

**Comment by lirantal** [comment]
thanks @filippolauria looks like a good change to run some updates on the libs for the whole project 👏👏👏

@MiguelVis can you take the branch here for a test drive?

**Comment by MiguelVis** [comment]
> thanks @filippolauria looks like a good change to run some updates on the libs for the whole project 
> 
> @MiguelVis can you take the branch here for a test drive?

Ok @lirantal  , I'll do some checks asap. Looks very well at first impression.

@filippolauria  have you finished adding commits? Thanks.

**Comment by filippolauria** [comment]
@MiguelVis this is the last one.
Thank you both

**Comment by lirantal** [comment]
Landing this to fix #204

**Pull request merged**